index.html

```diff
@@ -2,69 +2,120 @@
 </head>
 <body>
 	<main>
-		<div>
-			<div>
-				<div>
-					<div></div>
-					<a href=""></a>
+		<div class="sign-up" style="height: 60px;">
+			<div class="s-u-section">
+				<div class="s-u-left">
+					<div class="accnt-msg">Already have an Intuit account?</div>
+					<div class="accnt-msg">I forgot my User ID or Password</div>
 				</div>
-				<div>
-					<button>
-						<span>
+				<div class="s-u-right">
+					<button class="btn">
+						<span class="lock-img"></span> Sign In
 							
-						</span>
+						
 					</button>
 				</div>
 			</div>
 		</div>
 
-		<header>
-			<div>
-				<a href=""></a>
+		<header class="header-style">
+			<div class="logo-container">
+				<a href="" class="title-link">
+					
+				</a>
+				<img src="img/logoIntuit.png" class="logo-intuit">
 			</div>
 		</header>
 
 		<section>
 			<div>
 				<div>
 					<div class="icons">
-						<header>
-							<div>
+						<header class="header-style">
+							<div class="icons-container">
 								<a href=""></a>
-								<ul>
-									<li></li>
-									<li></li>
-									<li></li>
-									<li></li>
-									<li></li>
-									<li></li>
-									<li></li>
+								<ul class="form-icons">
+									<li><img src="img/intuit-mint-logo.png" style="padding: 1px;"></li>
+									<li class="icon-description">mint</li>
+									<li><img src="img/quickbooks-logo.png" style="padding-top: 1px;"></li>
+									<li class="icon-description">quickbooks</li>
+									<li><img src="img/turbotax-icon.png" style="padding-top: 1px;"></li>
+									<li class="icon-description">turbotax</li>
 								</ul>
 							</div>
 						</header>
 					</div>
-					<div>
-						<div></div>
-						<div></div>
-						<hr>
-						<section></section>
-					</div>
-					<footer>
-						<div>
-							<div>
+					<div class="form-area">
+						<section>
+							<header class="form-header">
+								<span class="create-acnt">Create An Intuit Account</span><br><br>
+								<span class="create-acnt-description">One account for everything Intuit,
+									 including Mint. </span>
+							</header>
+							<form>
+								<div>
+									<label class="form-label">Email address
+										<br>
+										<input type="email" name="email" class="form-input" required>
+									</label>
+								</div>
+								<br><br>
+								<div>
+									<label class="form-label">Phone (recommended)
+										<input type="text" name="phone" class="form-input" required>
+									</label>
+									<p class="call-rates">Standard call, messaging or data rates may apply.</p>
+								</div>
+								<br><br>
 								<div>
-									<div></div>
-									<ul></ul>
+									<label class="form-label">Password
+										<input type="password" name="password" class="form-input" required>
+									</label>
 								</div>
+								<br><br>
+								<div class="submit-button">
+									<button type="submit" class="btn-2">
+										<span>
+											<img class="lock-img" src="img/lock.png">
```

Comment:
Please see [HTML Checker](https://validator.w3.org/) and [this Comment]() on `<img>` tag `alt`.